src/pages/docs/chat/react-ui-components/components.mdx

```diff
@@ -0,0 +1,570 @@
+
+## Avatar <a id="avatar"/>
+
+The `Avatar` component displays a user or room avatar with fallback to initials when no image is available.
+
+### Props
+
+| Prop | Description |
+|------|-------------|
+| `src` | URL to the avatar image |
+| `alt` | Alternative text for the avatar image, also used for generating initials if needed |
+| `color` | Background color for the avatar when no image is provided (uses Tailwind CSS color classes) |
+| `size` | Size of the avatar |
+| `initials` | Custom initials to display when no image is available |
+| `onClick` | Click handler for the avatar |
+
+### Usage
+
+<Code>
+```react
+import { Avatar } from 'ably-chat-react-ui-components';
+
+// Basic usage
+<Avatar
+  src="https://example.com/avatar.jpg"
+  alt="John Doe"
+  size="md"
+/>
+
+// With fallback to initials
+<Avatar
+  alt="John Doe"
+  color="bg-blue-500"
+  size="lg"
+/>
+
+// With custom initials
+<Avatar
+  alt="John Doe"
+  initials="JD"
+  color="bg-green-500"
+  size="xl"
+/>
+
+// With click handler
+<Avatar
+  alt="John Doe"
+  onClick={() => console.log('Avatar clicked')}
+/>
+```
+</Code>
+
+### Integration with AvatarProvider
+
+The Avatar component is designed to integrate with the `AvatarProvider` to display consistent avatars across your application, but it also works independently. The `AvatarProvider` can be used to manage avatar generation and caching for users and rooms.:
+
+<Code>
+```react
+import { Avatar, useUserAvatar } from 'ably-chat-react-ui-components';
+const { userAvatar } = useUserAvatar({
+    clientId: 'user-123',
+    displayName: 'John Doe',
+  });
+
+<Avatar
+  src={userAvatar?.src}
+  alt={userAvatar?.displayName}
+  color={userAvatar?.color}
+  initials={userAvatar?.initials}
+/>
+```
+</Code>
+
+## ChatMessage <a id="chat-message"/>
+
+The `ChatMessage` component displays an individual chat message with interactive capabilities. It allows users to view, edit, delete, and react to messages.
+
+### Features
+
+- Message content display with sender avatar
+- Edit/delete functionality for own messages with confirmation dialogs
+- Emoji reactions system with picker and toggle functionality
+- Status indicators (edited, deleted)
+- Basic ARIA support (role, aria-label)
+- Hover tooltips showing sender information
+
+### Props
+
+| Prop | Description |
+|------|-------------|
+| `message` | The Ably Chat message object used to display the message content |
+| `onEdit` | Callback triggered when the user saves an edited message |
+| `onDelete` | Callback triggered when the user confirms message deletion |
+| `onReactionAdd` | Callback triggered when a user adds an emoji reaction to the message |
+| `onReactionRemove` | Callback triggered when a user removes their emoji reaction from the message |
+| `className` | Additional CSS class names to apply to the message container |
```

Comment:
I think we should have a separate page, or section at the top of this page, that calls out the approach to styling components, generally